graph/builder.go

```diff
@@ -0,0 +1,57 @@
+package graph
+
+import (
+	"sync"
+	"sync/atomic"
+)
+
+// Config holds the configuration required by the Builder.
+type Config struct{}
+
+// Builder builds and maintains a view of the Lightning Network graph.
+type Builder struct {
+	started uint32 // To be used atomically.
```

Comment:
Shouldn't we start using `atomic.Uint32` instead?

Reply:
Or `atomic.Bool`?

Reply:
also added a commit at the end to convert the other Builder members to use atomic.Unit*

Reply:
updated to `atomic.Bool` 👍